box2d3-wasm/README.md

```diff
@@ -1,20 +1,67 @@
-# Box2d3-wasm
+# box2d3-wasm
 
-Let's try and use box2d v3 on the web.
+Let's try and use [Box2D v3][] on the web via [wasm][]!
 
 ```bash
 git clone --recurse-submodules https://github.com/Birch-san/box2d3-wasm.git
 ```
 
-## Building
+## Pre-Requisites
 
-Install [Emscripten](https://emscripten.org/docs/getting_started/downloads.html).
+Install [Node.js and npm](https://nodejs.org/en/download/).
+
+emscripten seems to go bang if you PATH a local typescript, so I had to make it global.
 
 ```bash
-# emscripten seems to go bang if you PATH a local typescript, so I had to make it global
 npm i -g typescript
+```
+
+Next, install [Emscripten](https://emscripten.org/docs/getting_started/downloads.html).
+
+On macOS, you can use [Homebrew](https://brew.sh/) to install it.
+
+```bash
+brew install emscripten
+```
+
+## Build
+
+Run the following commands:
+
+```bash
 FLAVOUR=simd TARGET_TYPE=Debug ./shell/0_build_makefile.sh
 emmake make -j8 -C cmake-build
 FLAVOUR=simd TARGET_TYPE=Debug ./shell/1_build_wasm.sh
+```
+
+## Run Demos
+
+Install the "local-web-server" package globally (or locally in the `demo` directory).
+
+```bash
+npm i local-web-server -g
```

Comment:
there's no need to install local-web-server globally; it's already a devDependency of demo, so you can just install demo's dependencies. you're gonna need to yarn install the demo either way to get the npm link to happen

Reply:
ah okay, so yarn is necessary then. I can edit that.

Reply:
just fixed this